### DISCUSSION TITLE
品牌就是“黑話”

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1352] [Ideas]
當我們用力抽象，就會發現，產業資本同樣有一個“中介導致流通，流通生成差異，差異導致剩餘”的邏輯。

商業資本中貨幣中介生成了不同價值的同一商品，僱傭勞動的資本家也中介生成了作為同一人的生產者和消費者。同一個商品，能在不同的價值體系中流通，獲得差異，形成了剩餘；同一個具體的人，被當成勞動力商品時，也能在不同的價值體系——被購買的勞動力商品與購買商品的消費者——中產生差異，形成剩餘。

正如馬克思所說：

- 「把資本從支配關係中區別出來的，正是勞動者作為消費者及交換價值設定者而相對於資本的東西，而成為貨幣持有者的形態，貨幣形態上流通的單純起點——無限多的起點之一，在此，勞動者作為勞動者的規定性消失了。」《馬克思資本論草稿集》日文版第2卷，轉引自柄谷行人《跨越性批判》

此處我們要回答那個非常重要的問題：在最抽象的意義上，利潤的本質是什麼？它從何而來。

答案是，利潤是一種剩餘。同一個對象，在兩個價值體系之間運動，導致該對象的虛假同一性穿幫，顯現出差異，剩餘（利潤）就是伴隨著這個差異生成運動而排泄出來的東西。

這個命題幾乎不是人話，但卻能幫我們從一個特別的角度重新解釋很多經濟現象。我會掰開揉碎了和你說清楚。

通過前面的內容，「商品在兩個價值體系之間的運動」的意思大家應該清楚了，但什麼是「對象的虛假同一性」？

此處就需要用到語言系統的類比了。

先回顧前面的知識點，中介是如何導致流通產生的？它必須將所有對像都置於同一個價值序列之中。在這個序列中，我家的蘋果和你家的蘋果是同一的，都是作為商品的蘋果，如此一來，蘋果才能作為一個“類”與其他商品建立關係，進入所謂的“商品價值體系”。

這與語言交流的原理是一樣的，蘋果這個詞必須作為統一的範疇，才能被言說和理解。交易和交流之所以能進行，是因為作為貨幣價值形式和語言的中介系統能抹去具體對象的差異，建立抽象的一般對象。

問題是，這種同一性和一般性是虛假的。事實上，差異非但沒有被抹去，反而是被囊括了，要理解這件事，我們必須求助於符號學的智慧。

辯證法那期對索緒爾的引用，本期仍然適用：「所以在任何情況下，我們所看到的都不是預先規定了的觀念，而是來自於系統的價值。我們說價值相當於概念，其意思是，概念純粹是表示差異的，不能根據其內容從正面確定它們，只能根據它們與系統中其他成員的關係，從正面確定它們。它們確切的特徵是它們「不是」別的東西」索緒爾《普通語言學教程》

此處，請回顧前面對《資本論》的引用「一個商品，如麻布，處於能與其他一切商品直接交換的形式，是因為而且只是因為其他一切商品都「不是」處於這種形式……它的價值相對地表現在其他一切商品體的無限的系列上」

索緒爾原文裡的「價值」，指的是「語言符號的內涵」，在今天的語境下意味深長，他說「概念純粹是表示差異的，它表達了來自系統的價值」，我們說「商品的價值也來自差異，取決於它在整個商品無限系列中的相對位置」。

「兩個價值體系之間的運動」導致「對象的虛假同一性」穿幫，在語言實踐中十分常見：兩個人通過交流，發現彼此說的同一個詞居然有完全不同的意思，「意義」、「價值」、「友情」、「成功」，在不同人的“概念體系”中意涵完全不同。

在現實生活中，我們會用很多修飾、限定和定義來盡可能地縮小這種差異。 「你說的成功，只是世俗意義上的成功」，「你說的價值，是一種主觀判斷」等等，我們當然可以繼續追究「世俗」和「主觀」在兩人心中的差異，如此循環，永無止境。

在這個意義上，交流（流通）就是一種為了消滅概念差異而生產更多差異概念的矛盾運動。一個學科只要變成專業研究領域，交流足夠充分，就會像哲學這樣充滿了專業術語和黑話，因為它需要在極精微的層次上區分一個概念和另一個概念，所以語言的發展必然是一個“越描越黑”的過程。

交流之所以是永無止境的矛盾運動，是因為差異永遠是不可被徹底描述的，我們永遠覺得有些東西沒被表達出來，總是差那麼一點意思，正是“這點意思”，驅動著我們不停地運動。這個永遠逃避描述和捕捉的東西，就是「剩餘」。

現在我們來探討剩餘邏輯和商品利潤的關係。

在商業資本中，同一個商品，在價值體系A中，是「生產於中國的茶」，在體系B中，是「中國茶」，正因為中國茶不完全是一種茶，它的內在有某種多於茶，但又說不清楚是什麼東西。就像LV包包不僅是一個收納容器，它的內涵多於一個包包，包含了某種說不清楚是什麼東西。

我們會試圖描述這些「多出來」的東西，“身份的象徵”，“階級的證明”，“文化的符號”……但總是不止於此，也必須不止於此。一旦我們說「因為LV包包是身份的象徵，所以貴」的時候，它反而會被祛魅，讓人失去慾望，真正維持利潤的，是一種「否定性導致的絕對剩餘」，是一個「無法解開的謎語」。

包含著利潤的神秘商品，必須被這樣表述：「喝中國茶不僅是上流階級的自我表達，它還是……」「LV包包不僅是身份的象徵，它還是……」，「它還是」後面的省略號，是謎之利潤的真正來源。而當兩個價值體系趨於同一，當中國茶就是生產於中國的茶，LV包包就是包包的時候，差異空間消失了，迷之剩餘也就消失了。

在此，我們能為「價值」下一個「剩餘式」的定義：價值是「因為沒人知道它是什麼，所以人人都覺得它重要的東西」。

在語言活動中，同一語詞概念的細微差異，需要限定詞來確認，同一商品的利潤差異，也需要通過限定和修飾來強調——在這個意義上，「品牌」就是那個限定詞。

比如，這個星球上最值錢的品牌是「蘋果」，你去問一個資深果粉，「蘋果」這個品牌意味著什麼，特立獨行、反叛、品味、不妥協、創造力、嬉皮精神，他能給你寫篇萬字長文，你以為這就是蘋果品牌價值的體現？不，真正能表現蘋果品牌價值的，是另一個果粉在長文下的留言：「寫得不錯，但蘋果產品所蘊含的精神，遠不止於此。」

請你務必體會這句話中「不止於此」來帶的優越感和權力——不可消除的剩餘必須以一種相當微妙的形式介入人類關係模式，它才能充分發揮效力。

所以，品牌不是別的，它是商品世界的“黑話”——圈內人用它標榜自身，圈外人對此不明覺厲。一個哲學小白看哲學書，就像一個非洲部落的人進了發達國家的超市，首先是震撼，我原來的世界居然如此匱乏；然後是困惑，難道需要這麼多商品（詞語）才能正常生活（交流）嗎；最終會順從，管不了那麼多，讓我也豐盛起來吧，我要讓他人羨慕我，就像我羨慕這些豐盛一樣。

有些人上街，恨不能全是上下全是LOGO，有些人上網，恨不得通篇都是黑話大詞，兩者的心態是一樣的。在精神現象世界，品牌崇拜與黑話迷戀的原理如出一轍，可以歸為同一種愚蠢。

**Comment by hardtagottaname** [comment]
筆者講述了產業資本和商業資本在流通中生成差異，從而產生剩餘（即利潤）。我曾在相關書籍中讀過，流通和中介的邏輯類似於語言交流中的統一範疇和一般對象。然而，我認為這種統一和一般是虛假的，因為它們掩蓋了對象的差異。筆者又提到，通過把商品置於同一個價值序列中，中介將所有對象統一起來，使它們能夠作為“類”進入“商品價值體系”。我覺得這與語言交流中的原理相似，因為蘋果這個詞必須作為統一的範疇才能被言說和理解。

**Comment by Hahahahbla** [comment]
你的文章深入探討了資本主義中的利潤生成機制，並通過語言學的類比進一步闡述了商品價值和品牌的作用。這種跨領域的分析方法不僅豐富了對經濟現象的理解，也提供了一個新的視角來思考資本主義體系中的問題。

**Comment by NcuJerryKao** [comment]
這篇文章探討了產業資本的邏輯和利潤的本質，並涉及到語言系統的類比。以下是幾個建設性建議，幫助提升文章的結構和可讀性：
1. 引言和背景介紹：
- 引言部分可以更清晰地介紹文章的主題和目的，讓讀者明白接下來討論的焦點。例如，可以簡要說明產業資本和商業資本的基本概念以及文章將要探討的問題。
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點或問題。例如：
1. 引言：介紹主題和目的。
2. 產業資本的基本邏輯：解釋「中介導致流通，流通生成差異，差異導致剩餘」的邏輯。
3. 商業資本的具體案例：討論貨幣中介和僱傭勞動的具體運作方式。
4. 馬克思理論的引用和解釋：引述和解釋馬克思的觀點。
5. 利潤的本質：深入探討利潤的來源和意義。
6. 語言系統的類比：使用語言系統來解釋「虛假同一性」和「差異」。
7. 品牌的作用和意義：討論品牌在商品價值中的作用。
8. 結論：總結文章主要觀點。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我們將探討商業資本中的貨幣中介如何影響商品價值。」或「為了更好地理解這一點，我們可以借助語言系統的類比。」
4. 簡化和澄清語言：
- 某些段落中的語言較為複雜，可以考慮簡化和澄清。例如，「產業資本同樣有一個‘中介導致流通，流通生成差異，差異導致剩餘’的邏輯。」可以改為「產業資本的運作過程中，中介角色引發了商品流通，而流通過程中產生的價值差異則形成了利潤。」
5. 增加具體例子：
- 增加具體的實例來支持觀點。例如，在討論品牌價值時，可以引用具體的品牌案例，如蘋果公司的品牌價值如何體現其產品的差異性。
6. 圖表和引用：
- 若有可能，可以加入圖表或圖示來幫助解釋一些抽象概念。此外，引用一些具體的研究或數據來支持觀點會增加文章的說服力。
7. 結論強化：
- 在結論部分，再次強調文章的主要觀點，並總結文章的重要結論。這有助於讀者記住文章的核心思想。
8. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
通過這些建議，文章將變得更加結構化、清晰且易於理解，讀者也會更容易抓住文章的核心思想。

**Comment by titan92601** [comment]
我認為這個是產業資本中的流通、差異和利潤問題。透過中介，商品在不同價值體系中流動，形成剩餘。利潤的本質是差異，源於虛假同一性。這種流動使得交流不斷進行，而品牌成為差異的表達。資本主義經濟中，利潤來自於無法被完全描述的剩餘。

**Comment by sandyho0307** [comment]
我認為商品的中介轉換和貨幣的流通功能創造了商品價值的差異，這種差異讓原本同一的商品或服務在不同市場中得以獲得更高的價值認同。這種差異化不僅是市場經濟的基本運作模式，也是企業創新和品牌建立的重要策略。
